docs/manual/01_install/02_macOSX.md

```diff
@@ -1,49 +1,113 @@
 ---
-sidebar_label: Mac OS X
+sidebar_label: macOS
 ---
-# Installing on Mac OS X
 
-1. Go to the root folder where Ethereal Engine is stored and run
+# Installing on macOS
+
+## Introduction
+
+This guide details the installation process for the iR Engine on macOS devices.
+
+## Prerequisites
+
+Before starting the installation, ensure you have the following tools installed on your system:
+
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+- [nvm (Node Version Manager)](https://github.com/nvm-sh/nvm)
+- [Homebrew](https://brew.sh/)
+- [Python 3](https://www.python.org/downloads/)
+- [pip](https://pip.pypa.io/en/stable/installation/)
+
+## Installation instructions
+
+Follow these steps to set up the iR Engine on your macOS device.
+
+### Step 1: Prepare your project
+
+First, generate a copy of the project by cloning the repository and navigate to the project directory:
+
+1. Open your terminal
+2. Clone the repository by running:
+
+   ```bash
+   git clone https://github.com/EtherealEngine/etherealengine-docs.git
```

Comment:
Found a couple of typos:

```bash
git clone https://github.com/EtherealEngine/etherealengine.git
cd etherealengine
```
You are accidentally giving instructions to run the documentation website :see_no_evil:

Reply:
Great catch! Yeah, my terminal is full of commands that bounce back and fort between the two. I'll fix this right away.